lib/resources/api_base_helper.dart

```diff
@@ -66,3 +67,59 @@ class ApiBaseHelper {
     }
   }
 }
+
+/*
+Interceptor that modify API Request by adding authentication information to the request.
+ */
+class AuthInterceptor implements InterceptorContract {
```

Comment:
Can we move this class to its own separate file?

Reply:
Sure. We have two interceptors.  Can i move the AuthInterceptor and LogInterceptor to interceptor.dart file or seperately?

Reply:
Separate dart files for each class, in an interceptor folder preferably.

Reply:
Done the changes.